frontend/src/locales/de.json

```diff
@@ -96,7 +105,9 @@
       "add_work":"neuer Gemeinschaftsbeitrag"
     },
     "profil": {
-      "transactions":"transactions",
+      "edit": {
+
+      },
```

Comment:
empty?? Not present in the English locale

Reply:
ok